packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react'
+import PopperJS from 'popper.js'
+import { Ref } from '@stardust-ui/react-component-ref'
+
+import { getPlacement, applyRtlToOffset } from './positioningHelper'
+import { Alignment, Position } from './index'
+
+export interface PositionCommonProps {
+  /** Alignment for the component. */
+  align?: Alignment
+
+  /** Offset value to apply to rendered component. Accepts the following units:
+   * - px or unit-less, interpreted as pixels
+   * - %, percentage relative to the length of the trigger element
+   * - %p, percentage relative to the length of the component element
+   * - vw, CSS viewport width unit
+   * - vh, CSS viewport height unit
+   */
+  offset?: string
+
+  /**
+   * Position for the component. Position has higher priority than align. If position is vertical ('above' | 'below')
+   * and align is also vertical ('top' | 'bottom') or if both position and align are horizontal ('before' | 'after'
+   * and 'start' | 'end' respectively), then provided value for 'align' will be ignored and 'center' will be used instead.
+   */
+  position?: Position
+
+  /**
+   * Array of conditions to be met in order to trigger a subsequent render to reposition the elements.
+   */
+  positioningDependencies?: any[]
```

Comment:
```suggestion
  positioningDependencies?: React.DependencyList
```

nit: We can use `DependencyList` type to comply

Reply:
done, thanks